scss/_patterns_tabs.scss

```diff
@@ -3,6 +3,7 @@
 @mixin vf-p-tabs {
   .p-tabs {
     border-radius: 0;
+    color: $colors--theme--text-default;
```

Comment:
This should not be needed I think.

Once the #4970 is merged you should be able to change background and text colour of body element, and tabs could simply inherit it via CSS without specifying it (I think).